src/types.rs

```diff
@@ -883,3 +883,57 @@ where
     }
     deserializer.deserialize_any(HeightVisitor)
 }
+
+#[derive(Clone, Debug, PartialEq, Serialize, Default)]
+pub struct WalletCreateFundedPsbtOptions {
+    #[serde(rename = "fee_rate", skip_serializing_if = "Option::is_none")]
+    pub fee_rate: Option<f64>,
+    #[serde(rename = "lock_unspent", skip_serializing_if = "Option::is_none")]
+    pub lock_unspent: Option<bool>,
+    #[serde(rename = "conf_target", skip_serializing_if = "Option::is_none")]
+    pub conf_target: Option<u16>,
+    #[serde(rename = "replaceable", skip_serializing_if = "Option::is_none")]
+    pub replaceable: Option<bool>,
+}
+
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct WalletCreateFundedPsbt {
+    pub psbt: String,
+    #[serde(deserialize_with = "deserialize_bitcoin")]
+    pub fee: Amount,
+    #[serde(rename = "changepos")]
+    pub change_pos: i32,
+}
+
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct WalletProcessPsbt {
+    pub psbt: String,
+    pub complete: bool,
+}
+
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct FinalizePsbt {
+    pub psbt: Option<String>,
+    pub hex: Option<String>,
+    pub complete: bool,
+}
+
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct GetAddressInfo {
+    pub address: String,
+    #[serde(rename = "ismine")]
+    pub is_mine: Option<bool>,
+    #[serde(rename = "iswatchonly")]
+    pub is_watchonly: Option<bool>,
+    pub solvable: Option<bool>,
+}
+
+#[derive(Clone, Debug, PartialEq, Serialize)]
+pub struct ListUnspentQueryOptions {
+    #[serde(rename = "minimumAmount", skip_serializing_if = "Option::is_none")]
+    pub minimum_amount: Option<f64>,
+    #[serde(rename = "maximumAmount", skip_serializing_if = "Option::is_none")]
+    pub maximum_amount: Option<f64>,
+    #[serde(rename = "maximumCount", skip_serializing_if = "Option::is_none")]
+    pub maximum_count: Option<u32>,
+}
```

Comment:
NACK: all these `rename_with` can be replaced by a single `#[serde(rename_all = "camelCase")]` in the `struct`.
See: https://serde.rs/attr-rename.html

```suggestion
#[derive(Clone, Debug, PartialEq, Serialize)]
#[serde(rename_all = "camelCase")]
pub struct ListUnspentQueryOptions {
    #[skip_serializing_if = "Option::is_none")]
    pub minimum_amount: Option<f64>,
    #[skip_serializing_if = "Option::is_none")]
    pub maximum_amount: Option<f64>,
    #[skip_serializing_if = "Option::is_none")]
    pub maximum_count: Option<u32>,
}
```